tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -75,12 +75,20 @@ inline SPTR GetParamSptr(int dpdsc, int i) {
 #define DW_TAG_vector_type 0x103
 #endif
 
-const int DIFLAG_ARTIFICIAL = 1 << 6;
-const int DIFLAG_ISMAINPGM = 1 << 21;
+static int DIFLAG_ARTIFICIAL;
+static int DIFLAG_ISMAINPGM;
 static int DIFLAG_PURE;
 static int DIFLAG_ELEMENTAL;
 static int DIFLAG_RECUSIVE;
 
+static int DISPFLAG_LOCALTOUNIT;
+static int DISPFLAG_DEFINITION;
+static int DISPFLAG_OPTIMIZED;
+static int DISPFLAG_PURE;
+static int DISPFLAG_ELEMENTAL;
+static int DISPFLAG_RECUSIVE;
+static int DISPFLAG_ISMAINPGM;
```

Comment:
These flags should all be `const` since they do not change in prior versions of LLVM IR. (They didn't exist or were not used if they did, so there is no backward compatibility issue.)